src/core-concepts-in-embedded-application/logging.md

```diff
@@ -0,0 +1,118 @@
+# Logging
+
+​Logging is a crucial aspect of embedded systems development, providing visibility into the system's behavior and aiding in debugging and monitoring. In the Rust ecosystem for ESP devices, two prominent logging frameworks are commonly used: the [log] crate and [defmt]. The [esp-println] crate serves as a bridge, facilitating logging through these frameworks on ESP devices.​
```

Comment:
```suggestion
​Logging is a crucial aspect of embedded systems development, providing visibility into the system's behavior and aiding in debugging and monitoring. In the Rust ecosystem for ESP devices, two prominent logging frameworks are commonly used: the [log] crate and [defmt].
```

Whilst that statement is true to some extent, it makes it seem like users of prob-rs + defmt need esp-println, but they don't. You cover this below anyways, so lets remove it from here.